crates/pop-drink/src/error.rs

```diff
@@ -0,0 +1,258 @@
+//! A set of errors used for testing smart contracts.
+
+use std::fmt::Debug;
+
+pub use drink::{
+	pallet_assets::Error as AssetsError, pallet_balances::Error as BalancesError,
+	pallet_contracts::Error as ContractsError,
+};
+pub use pop_api::primitives::v0::*;
```

Comment:
Not sure this export should be here, but rather under the runtime/version module instead.

Reply:
Resolved